packages/tools/src/tools/annotation/ArrowAnnotateTool.ts

```diff
@@ -801,54 +812,56 @@ class ArrowAnnotateTool extends AnnotationTool {
         continue;
       }
 
-      const options = this.getLinkedTextBoxStyle(styleSpecifier, annotation);
-      if (!options.visibility) {
-        data.handles.textBox = {
-          hasMoved: false,
-          worldPosition: <Types.Point3>[0, 0, 0],
-          worldBoundingBox: {
-            topLeft: <Types.Point3>[0, 0, 0],
-            topRight: <Types.Point3>[0, 0, 0],
-            bottomLeft: <Types.Point3>[0, 0, 0],
-            bottomRight: <Types.Point3>[0, 0, 0],
-          },
-        };
-        continue;
-      }
+      if (this.configuration.renderTextBox) {
```

Comment:
if (!this.configuration.renderTextBox){ return renderStatus} 

instead of indenting the code